subscription-manager.spec

```diff
@@ -276,28 +252,16 @@ from the Red Hat entitlement platform; contains subscription-manager and
 product-id plugins.
 %endif
 
-# This redefinition of debuginfo package has to be here, because we
-# need to solve the issue described in this BZ:
-# https://bugzilla.redhat.com/show_bug.cgi?id=1920568
-# We need to obsolete old dnf-sub-man-plugin-debuginfo RPM
-%package -n libdnf-plugin-subscription-manager-debuginfo
-Summary: Debug information for package libdnf-plugin-subscription-manager
-Obsoletes: dnf-plugin-subscription-manager-debuginfo < 1.29.0
```

Comment:
This is correct change.

However, I can see this line `Obsoletes: dnf-plugin-subscription-manager < 1.29.0` one more time in the spec file (definition of `subscription-manager` RPM). Shouldn't we also remove another occurrence of this line?

Reply:
that one is to migrate from `dnf-plugin-...` to `libdnf-plugin-...`; I think that might be obsolete too, I decided to leave it around for this round of cleanups